recipes/libtommath/recipe.yaml

```diff
@@ -0,0 +1,69 @@
+context:
+  version: "1.3.0"
+
+package:
+  name: libtommath
+  version: ${{ version }}
+
+source:
+  url: https://github.com/libtom/libtommath/archive/v${{ version }}.tar.gz
+  sha256: 6d099e93ff00fa9b18346f4bcd97dcc48c3e91286f7e16c4ac5515a7171c3149
+
+build:
+  number: 0
+  skip:
+    - osx
+  script:
+    - if: unix
+      then:
+        - mkdir build
+        - cd build
+        - cmake -DCMAKE_INSTALL_PREFIX=$PREFIX -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=OFF -DCMAKE_POSITION_INDEPENDENT_CODE=ON ..
```

Comment:
I think we actually should prefer dynamic libraries. If you really want to package only static, then read [CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md)